### PR TITLE
[PSDK-356] Mass Payout Quickstart Template

### PR DESCRIPTION
**Pull request by John-peterson-coinbase**
### What changed? Why?
- Updates Quickstart template README so that the commands work successfully 
- Adds Mass Payout template  (`npm run start-mass-payout`)

### CODE REVIEW

**Comment by cb-heimdall** [comment]
<h1>🟡 Heimdall Review Status</h1>
<table>
<tr>
<th>Requirement</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Reviews</td>
<td>
🟡
<code>0/1</code>
</td>
<td>
<details>
<summary>Denominator calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>.codeflow.yml</code>
</td>
<td>1</td>
</tr>
<tr>
<td>
Additional review requirements
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>CODEOWNERS</code>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>0</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>1</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>

File: quickstart-template/README.md [hunk 1]
```diff
@@ -2,10 +2,22 @@
...
+
+To set up the template, run the following commands:
+```bash
+npm install
+npm run start
+```
+
+This command will create a developer-custodial wallet, deposit testnet funds to it and perform a transfer to another wallet.
+
+## Mass Payout
```
Comment (jazz-cb) [inline quickstart-template/README.md:15]: could you also add a section for trade?

File: quickstart-template/mass-payout.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
+import { Coinbase } from "@coinbase/coinbase-sdk";
+import { createArrayCsvWriter } from "csv-writer";
+import os from "os";
+import fs from "fs";
+import { parse } from "csv-parse";
+
+// CREATE RECEIVING WALLETS
```
Comment (jazz-cb) [inline quickstart-template/mass-payout.js:7]: should these be in CAPS?

File: quickstart-template/README.md (hunk 1 above)
Comment (John-peterson-coinbase) [reply quickstart-template/README.md:25]: Added in most recent commit

File: quickstart-template/mass-payout.js (hunk 2 above)
Comment (John-peterson-coinbase) [reply quickstart-template/mass-payout.js]: Resolved in most recent commit

**Review by jazz-cb — APPROVED** [review]

**Pull request merged**